logic.js

```diff
@@ -0,0 +1,122 @@
+// const tweeter = () => {
```

Comment:
Remove zombie code (any unused code that is sitting in comments). Just makes code noisier.